docs/quickstart.rst

```diff
@@ -60,38 +23,27 @@ users will start running within their own green gevent thread.
 
 .. code-block:: python
 
-    wait_time = between(5, 9)
-
-Our class defines a ``wait_time`` function that will make the simulated users wait between 5 and 9 seconds after each task 
-is executed. 
-
-.. code-block:: python
-
-    @task(2)
-    def index(self):
-        self.client.get("/")
-        self.client.get("/ajax-notifications/")
+    @task
+    def hello(self):
+        self.client.get("/hello")
+        self.client.get("/world")
```

Comment:
Same here. I much prefer potential real world examples than very generic names.

Reply:
Hmm... I prefer using obviously made up names instead of names that could be interpreted as meaningful but aren't (such as /ajax-notifications/)

Reply:
I'm sure it could be improved further, but I think the old examples are worse (for the reasons stated above)

Reply:
Hm, in this case I don't think there's a big risk of people thinking those names have special meaning for locust.

Reply:
> I'm sure it could be improved further, but I think the old examples are worse (for the reasons stated above)

I'm sorry to say I don't agree :/

Reply:
Perhaps we could come up with different names that has no chance of misleading people into believing they might have special meaning, while still resembling some potential real world scenario?

Reply:
Maybe, but I dont have the energy to do it atm. Feel free to update as you see fit.

Reply:
Ok, sounds good! 👍